README.md

```diff
@@ -270,7 +287,9 @@ wayland-rs | [#187](https://github.com/Smithay/wayland-rs/pull/187) | libfuzzer
 ws-rs | [arithmetic overflow](https://github.com/housleyjk/ws-rs/pull/179) | libfuzzer | `arith`
 xi-editor | [issue/1303](https://github.com/xi-editor/xi-editor/issues/1303) | afl | `arith`  
 xml-rs | [#93](https://github.com/netvl/xml-rs/issues/93) | afl | `utf-8`
+xml-rs | [arithmetic overflow](https://github.com/netvl/xml-rs/issues/204) | libfuzzer | `arith`
 zip-rs | [arithmetic overflow](https://github.com/mvdnes/zip-rs/issues/40) | libfuzzer | `arith`
+zip-rs | [arithmetic overflow](https://github.com/zip-rs/zip/issues/234) | libfuzzer | `arith`
```

Comment:
The repo link here has changed (and the crate is called `zip`, not `zip-rs`), should I change both of these?